contracts/IBlockReward.sol

```diff
@@ -3,4 +3,5 @@ pragma solidity 0.4.24;
 
 interface IBlockReward {
     function addExtraReceiver(uint256 _amount, address _receiver) external;
+    function totalMintedCoins() public view returns (uint256);
```

Comment:
Please declare `mintedTotally` as it is intended to be. Please refer to https://github.com/poanetwork/poa-network-consensus-contracts/pull/197.

Reply:
Thanks, updated method name on interface and test contract implementation